Kanban_doreenbinwi@yahoo.com.md

```diff
@@ -0,0 +1,75 @@
+
+---
+
+## Q2: What is lead time and cycle time ? How can they be used in Kanban?
+
+**Difficulty:** `Senior`
+
+**Source:**
+
+https://teamhood.com/kanban-/lead-and-cycletime-kanban-metrics/
+
+**Answer:**
+
+Lead time is the average time it takes for the team to deliver a product to client from the time it was requested ( appeared in the backlog ). While cycle time is the average time it takes for the team to complete an item from the time they started working in it. They are both metrics that can be used in Kanban as follows:
+
+* Calculating the average lead time gives a good idea of the delivery speed and cycle time provides an indication of the team's speed Thus, following both and noting down changes can give you valuable insights into what is happening in your project now and what might be happening in the future.
+
+* Lead time tracks the whole process from the initial request to the product delivery. Meaning you have a good idea of how long the production of an average item is going to take and can give the client a more accurate guess of delivery.
+
+* Cycle time on the other hand focuses just on the production phase of the process. It aims to track and signal any issues in the production process focused only on manufacturing and testing can provide great insights into where your own process is lagging and could be improved. Could also signal that new roadblocks or issues have arisen.
+
+* Lastly using a combinaition of the Kanban metrics allows the team to react to changes and adapt more quickly for example, if the team notices an increase in the cycle time it is safe to assume that the lead time will also grow. Thus, an increase in either of the metrics could signal a new roadblock or issue in the later project phases and a sudden decline could mean tasks are of a smaller size or the process has become more effective. How ever for clients with sensitive delivery dates the team can then take action to fast track other processes outside manufacturing. Thus still managing to deliver the end result in a timely manner. More over such efforts could lead to finding and applying completly new ways of optimizing the process.
+
+---
+
+## Q3: What are the differences between Kanban and Scrum ?
+
+**Difficulty:** `Junior`
+
+**Source:**
+
+https://www.atlassian.com/agile/kanban/kanban-vs-scrum
+
+**Answer:**
+
+* Kanban is a framework used to implement agile and Devops software development. It requires real-time communication of capacity and full transparency of work. Work items are presented vissually on a Kanban board allowing team members to see the state of every piece of work at anytime.
+
+* scrum is a framework that helps teams work together. It encourages teams to learn through experiences, selforganize while work on a problem and reflect on their wins and losses to continually improve.
```

Comment:
Please fix the Uppercase in the paragraph.

Reply:
Hello Mr Alex sure right away.
Thank you.